doc/administrator/editing.rst

```diff
@@ -76,6 +78,45 @@ corresponding to this field is *Related Postgres table* in the admin interface.
 
     It is recommended to limit the editing to 30 editable layers per user.
 
+Configure layer editing
+-----------------------
+An overview of the metadata for the configuration of the editing can be found here:
+:ref:`administrator_administrate_metadata`.
+
+Registering edit time and user
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* To register the time of the change of an edited item, a column of type
+  ``timestamp without timezone`` is needed in the table of the layer. In
+  the configuration of the layer ``lastUpdateDateColumn`` has to be added
```

Comment:
This is also described in https://camptocamp.github.io/c2cgeoportal/2.6/administrator/administrate.html?highlight=lastupdatedatecolumn

Should it be removed from there, and a link provided there to this new section?